src/lib/select/select.ts

```diff
@@ -238,6 +235,14 @@ export class MatSelect extends _MatSelectMixinBase implements AfterContentInit,
   /** The scroll position of the overlay panel, calculated to center the selected option. */
   private _scrollTop = 0;
 
+  /**
+   * The max height of the select's overlay panel
+   * 256 value is define has follow:
```

Comment:
We try to avoid putting values in documentation, because it's easy for it to go out of sync.